js/main.js

```diff
@@ -38,11 +38,8 @@
     return this.ajax(this.apiUrl + '/user');
   };
 
-  GhApi.prototype.getRepoCommits = function(repoPath) {
-    var self = this;
-    return this.getRepoData(repoPath).then(function(repo) {
-      return self.ajax(self.apiUrl + '/repos/' + repoPath + '/commits/' + repo.default_branch);
-    });
+  GhApi.prototype.getRepoCommits = function(repoPath, branch) {
+    return this.ajax(this.apiUrl + '/repos/' + repoPath + '/commits/' + (branch || 'master'));
```

Comment:
I feel like it makes good since for the API of this method to provide some default.  I dislike required parameters when there is a potential default available.  If you disagree, I can change it.